build/charts/antrea/values.yaml

```diff
@@ -206,7 +206,10 @@ clientCAFile: ""
 # -- To explicitly enable or disable a FeatureGate and bypass the Antrea
 # defaults, add an entry to the dictionary with the FeatureGate's name as the
 # key and a boolean as the value.
-featureGates: {}
+featureGates:
+  SecondaryNetwork: true
+  EgressSeparateSubnet: true
+  ServiceExternalIP: true
```

Comment:
Are these temporary changes for local testing?  Don't forget to remove test codes If it's ready for review, or convert this PR into a draft one.

Reply:
done. reverted unrelated changes.